pkg/vpa/vpa.go

```diff
@@ -110,7 +112,7 @@ func (r Reconciler) ReconcileNamespace(namespace *corev1.Namespace) error {
 		return err
 	}
 
-	return r.reconcileControllersAndVPAs(namespace, vpas, controllers)
+	return r.reconcileControllersAndVPAs(namespace, vpas, controllers, r.ignoreControllerKind)
```

Comment:
This field is already contained in the reconciler, no need to pass it like this.

```suggestion
	return r.reconcileControllersAndVPAs(namespace, vpas, controllers)
```